toolkit/tools/pkg/imagecustomizerlib/defaultgrubutils.go

```diff
@@ -175,6 +175,42 @@ func GetDefaultGrubFileLinuxArgs(defaultGrubFileContent string, varName defaultG
 	return cmdLineVarAssign, args, insertAt, nil
 }
 
+// GetDefaultGrubFileLinuxArgsFromMultipleVars attempts to get Linux kernel command-line args from multiple
+// GRUB variables. It merges arguments from all found variables, allowing args to be distributed across
+// multiple GRUB variables like GRUB_CMDLINE_LINUX and GRUB_CMDLINE_LINUX_DEFAULT.
+//
+// Parameters:
+//   - defaultGrubFileContent: The content of the /etc/default/grub file
+//   - varNames: Variable names to check and merge arguments from
+//
+// Returns:
+//   - args: The merged list of kernel command-line args from all found variables
+//   - error: Error if parsing fails or no variables are found
+func GetDefaultGrubFileLinuxArgsFromMultipleVars(defaultGrubFileContent string, varNames ...defaultGrubFileVarName,
```

Comment:
There are only two variables. So, you could just embed the names within this function's implementation, instead of requiring callers to pass them in.